src/sorcha/ephemeris/simulation_parsing.py

```diff
@@ -21,7 +21,7 @@ def mjd_tai_to_epoch(mjd_tai):
     return epoch
 
 
-def parse_orbit_row(row, epochMJD_TDB, ephem, sun_dict, gm_sun):
+def parse_orbit_row(row, epochMJD_TDB, ephem, sun_dict, gm_sun, gm_total):
     orbit_format = row["FORMAT"]
 
     if orbit_format != "CART":
```

Comment:
@bernardinelli  and I think this should be  
`if orbit_format not in ['CART', 'BCART']`

Reply:
What is the second requested change?

Reply:
The ValueError - we should be putting that into the error log file and exiting out because something went wrong. I am not sure the error log is set up

Reply:
Someone else will need to do that.  I am not up to speed on the logging system.

Reply:
Logging is done.